dds_web/__init__.py

```diff
@@ -146,6 +150,14 @@ def create_app(testing=False, database_uri=None):
     # Initialize marshmallows
     ma.init_app(app)
 
+    # Initialize login manager
+    login_manager.init_app(app)
+
+    @login_manager.user_loader
+    def load_user(user_id):
+        # since the user_id is just the primary key of our user table, use it in the query for the user
+        return models.User.query.get(user_id)
```

Comment:
Do you think we need some SQLAlchemy Error handling here? Feel free to create a card about though.

Reply:
True, I can add that now.

Reply:
@alneberg Have added an extremely simple error handler above this with a comment on what more is needed there.